cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/util/OAuthUtils.java

```diff
@@ -85,4 +85,18 @@ public static OAuthRegisteredService getRegisteredOAuthService(final ServicesMan
         }
         return null;
     }
+
+    /**
+     * Url encode a value.
+     *
+     * @param value the value to encode
+     * @return the encoded value
+     */
+    public static String urlEncode(final String value) {
```

Comment:
We don't have any centralized `urlEncode` method currently and I have found several occurences here and there so we need one.
`CompressionUtils` doesn't seem the best option, I would prefer using the new `EncodingUtils`.